ifrs17-template/Report/ReportEyCases.ipynb

```diff
@@ -47,7 +47,17 @@
         {
             "cell_type": "code",
             "source": [
-                "#!import \"../Initialization/InitSystemorphToMemory\""
+                "#!import \"../Initialization/InitSystemorphUtils\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await InitializeDataNodesAsync(DataSource, pathToImport, new string []{\"FR\"});",
```

Comment:
pathToImport => explicitly write the path

Reply:
done